ugs-platform/ugs-platform-plugin-joystick/src/main/resources/resources/MessagesBundle.properties

```diff
@@ -0,0 +1,26 @@
+platform.plugin.joystick.activate = Activate joystick
```

Comment:
I'm still confused about how these work. Is the per-plugin message bundle overridden by the global message bundle?

Reply:
No you are right. I used the MessagesBundle as the module bundle before, but this doesn't work anymore with the latest version of Netbeans. I removed this.